### ISSUE TITLE
Trying to access array offset on value of type null

### ISSUE THREAD

**Issue by vikioki** [lavary/laravel-menu#284]
php -v
```
PHP 8.0.2 (cli) (built: Feb 14 2021 14:21:37) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.2, Copyright (c), by Zend Technologies
```

apache2 -v
```
Server version: Apache/2.4.41 (Ubuntu)
Server built:   2020-08-12T19:46:17
```

artisan --version
`Laravel Framework 8.28.1`

laravel ErrorException => "Trying to access array offset on value of type null"
```
Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
vendor/lavary/laravel-menu/src/Lavary/Menu/Menu.php:108
```
```
     * Loads and merges configuration data.

     *

     * @param string $name

     *

     * @return array

     */

    public function loadConf($name)

    {

        $options = config('laravel-menu.settings');

        $name = strtolower($name);

 

        if (isset($options[$name]) && is_array($options[$name])) {

            return array_merge($options['default'], $options[$name]);

        }

 

        -->> return $options['default'];

    }

 

    /**

     * Return Menu builder instance from the collection by key.

     *

     * @param string $key

     *

     * @return Builder

     */

    public function get($key)

    {

        return $this->collection->get($key);

    }

 

    /**
```

im fixed it like this -> from `return $options['default']` to `return (!empty($options["default"])) ?  $options['default'] : [];`

**Comment by dustingraham** [comment]
Thank you vikioki

@mertasan are you able to take a look at this? Appears related to your recent PR. Let me know. Thanks.

**Comment by mertasan** [comment]
@vikioki thanks for reporting this error. #285

**Comment by dustingraham** [comment]
Released/tagged v1.8.3
